packages/apollo-link-schema/src/schemaLink.ts

```diff
@@ -0,0 +1,57 @@
+import { ApolloLink, Operation, FetchResult, Observable } from 'apollo-link';
+
+import { print } from 'graphql/language/printer';
+import { graphql, GraphQLSchema } from 'graphql';
+
+export class SchemaLink extends ApolloLink {
+  public schema: GraphQLSchema;
+  public rootValue: any;
+  public context: any;
+
+  constructor({
+    schema,
+    rootValue,
+    context
+  }: {
+    schema: GraphQLSchema;
+    rootValue?: any;
+    context?: any;
+  }) {
+    super();
+
+    this.schema = schema;
+    this.rootValue = rootValue;
+    this.context = context;
+  }
+
+  public request(operation: Operation): Observable<FetchResult> | null {
```

Comment:
`forward` isn't implemented in the request method. My assumption is that not calling forward will break any chaining. Is that correct? 

```js
const consoleLink = new ApolloLink((operation, forward) => {
    // ---- vvvvvvv ---- needs to be preserved?
    return forward(operation).map((data) => { 
      // And this is where the gql call would be made?
      return data;  
    })
})
```